### PR TITLE
Fix -Wduplicated-branches found when build with gcc7 (and -Wduplicated-branches flags)

### PR DESCRIPTION
**Pull request by alagoutte**

### CODE REVIEW

File: src/airdecloak-ng.c [hunk 1]
```diff
@@ -988,17 +988,9 @@ int CFC_filter_signal() {
 						++nb_packets;
 						break;
 					}
-
-
-					if (_packet_elt_head->current->signal_quality - average_signal == 0) {
```
Comment (Mister-X-) [inline src/airdecloak-ng.c:993]: That information is lost (as well as the comment below). Comment should read something like:

If the signal quality is the same as the average signal, we're sure it's not a cloaked packet.
Regarding any other signal difference compared to the average signal, ... (the rest starting with "We could").

**Comment by Mister-X-** [comment]
Is there any speed difference due to the change in value in the simd code?

File: src/airdecloak-ng.c (hunk 1 above)
Comment (alagoutte) [reply src/airdecloak-ng.c:993]: OK i will update

**Comment by alagoutte** [comment]
>Is there any speed difference due to the change in value in the simd code?
How i can test ? (because there is typo and use the same value like John the Ripper)

File: src/airdecloak-ng.c (hunk 1 above)
Comment (alagoutte) [reply src/airdecloak-ng.c:993]: What do you prefer a amend this patch or a new commit ?

Comment (Mister-X-) [reply src/airdecloak-ng.c:993]: Looks good like that. All I wanted is to be able to get a diff with the updated changes.

**Comment by Mister-X-** [comment]
To test for speed, test with an older gcc, 6 and do a few runs, with and without the change.

File: src/airdecloak-ng.c (hunk 1 above)
Comment (alagoutte) [reply src/airdecloak-ng.c:993]: Ok !

**Comment by alagoutte** [comment]
Make some "quick" test and no found difference speed

**Comment by aircrack-ng** [comment]
Committed in r2909. Thanks.

**Pull request closed without merging**